ts/Extensions/Boost/BoostSeries.ts

```diff
@@ -670,6 +667,20 @@ function enterBoost(
     if (series.labelBySeries) {
         series.labelBySeries = series.labelBySeries.destroy();
     }
+
+    // Destroy existing points after zoom out
+    if (
+        series.type === 'scatter' &&
```

Comment:
In other parts of HC, we consistently use `series.is(type)` for series type checking, otherwise we will not capture series types that are inherited. In Boost we haven't done that change, mainly because we haven't had issues with it yet. But I think we should go for `series.is('scatter')`.